src/Command.js

```diff
@@ -175,7 +175,7 @@ export default class Command {
   }
 
   runValidations() {
-    if (!GitUtilities.isInitialized(this.execOpts)) {
+    if (!this.options.skipGitCheck && !GitUtilities.isInitialized(this.execOpts)) {
```

Comment:
I'd rather this was some sort of class property optionally overridden (kinda like the `otherCommandConfigs()` getter) when a command requires git. The base class has `get requiresGit() { return true; }`, and various commands that don't need git can opt-out by overriding the getter method.